src/python/pants/backend/jvm/targets/jar_library.py

```diff
@@ -77,7 +77,7 @@ def excludes(self):
     return self.payload.excludes
 
   @property
-  def exports(self):
+  def export_specs(self):
```

Comment:
Can't re-name an `:API: public`

Reply:
This isn't public, afaict because this method/property isn't marked public: https://www.pantsbuild.org/deprecation_policy.html ... the class is public, but that doesn't make all of its members public.

Reply:
Lines 81-83 just below are a doc comment with :API: public from what I can read in the GitHub diff ui. Maybe reading it wrong.

Reply:
Ah, you're right. It was public on `jar_library` but not on `jvm_target`. I'm going to violate the rules and just strip that, given that I expect it was marked this way simply because the other fields were.